examples/http-client-send-request-receive-response/http_client_send_request_receive_response.md

```diff
@@ -1,6 +1,6 @@
 # HTTP client - Send request/Receive response
 
-The HTTP client can be used to connect to and interact with an HTTP server. The client is instantiated with URL and uses resource method to make the network calls. The standard HTTP methods `get`, `post`, `put`, `patch`, `delete`, `head`, `options` are available as resource methods and can be invoked as same as invoking a remote method. To invoke an HTTP method, the relevant verb and the required arguments can be provided after the `->`. For `get` method, the verb is not explicitly needed since it will be defaulted.
+The `http:Client` interacts with an HTTP server. The client is instantiated with the service URL and it uses resource methods to send/receive requests/responses from the backend service. The standard HTTP methods `get`, `post`, `put`, `patch`, `delete`, `head`, and `options` are available as resource methods. A resource method can be invoked by providing the `resource path`, relevant resource method, and required arguments after the `->`. Since HTTP `get` is the default resource method, the accessor is not mandatory when accessing an HTTP `GET` resource.
```

Comment:
```suggestion
The `http:Client` interacts with an HTTP server. The client is instantiated with the service URL and uses resource methods to send requests to the backend service. The standard HTTP methods `get`, `post`, `put`, `patch`, `delete`, `head`, and `options` are available as resource accessors. A resource method invocation is done by providing the `resource path`, relevant `resource accessor`, and required arguments after the `->`. Since HTTP `get` is the default resource method, the accessor is not mandatory when invoking an HTTP `GET` resource.
```

Reply:
Fixed with [ee2c31ed2d6e7244a7858bb4bcebdf2385e21087](https://github.com/ballerina-platform/ballerina-distribution/pull/3764/commits/ee2c31ed2d6e7244a7858bb4bcebdf2385e21087)